app/controllers/reports/regions_controller.rb

```diff
@@ -61,6 +61,12 @@ def details
     @dashboard_analytics = @region.dashboard_analytics(period: @period.type,
                                                        prev_periods: 6,
                                                        include_current_period: true)
+    @chart_data = {
+      patient_breakdown: PatientBreakdownService.call(region: @region),
+      ltfu_trend: Reports::RegionService.new(region: @region,
```

Comment:
Looks like we should really make the LTFU data a separate piece so we don't need to retrieve all the other region related data with it. I suppose that will be a lot easier once we get https://github.com/simpledotorg/simple-server/pull/2014 merged.